libs/langchain/langchain/text_splitter.py

```diff
@@ -261,15 +261,21 @@ async def atransform_documents(
 class CharacterTextSplitter(TextSplitter):
     """Implementation of splitting text that looks at characters."""
 
-    def __init__(self, separator: str = "\n\n", **kwargs: Any) -> None:
+    def __init__(
+        self, separator: str = "\n\n", is_separator_regex: bool = False, **kwargs: Any
```

Comment:
hmm should we default to True so this doesn't change default behavior / is backwards compatible?